src/contacts/EditContact/EditContact.js

```diff
@@ -260,7 +253,7 @@ const EditContact = ({
                 >
                   <AddressInfo
                     dropdownVendorCategories={categories}
-                    dispatchChange={dispatchChange}
+                    change={change}
```

Comment:
actually, we don't need to pass this as props. it's can be used anywhere with hooks, `useForm`